### DISCUSSION TITLE
A new subscription plan

### DISCUSSION THREAD

**Discussion by ABDO-AR** [supabase/supabase#14009] [Feature Requests]
# Adding PAYG(Pay as you go) plan subscription

## Feature Explanation 

Supabase is the most powerful Backend tool, it's even better than Firebase in all things, but first issue is you need to add crashlytics logs like this one that Firebase had, secondly you need to include pay as you go plan for example if I want to only develope or test the project in a premium feature I have to pay at least $25/m and that's are not that good really.

## Additional context

I think Firebase is more famous and more used by people because that have pay as you go plan so I hope really you add that & thanks 🥰

**Comment by gregnr** [comment]
Thanks for the feedback @ABDO-AR! I'm going to convert this to a discussion - let's keep the conversation going there 😃

**Reply by ABDO-AR** [reply]
Thanks for replying 😊

**Comment by Ziinc** [comment]
The team is currently working towards supporting Open Telemetry for enabling such debugging scenarios.

For development or testing, we would suggest using the cli and the [local development setup](https://supabase.com/docs/guides/getting-started/local-development) for such scenarios, which provides the stack in a local environment.
